lib/floki/raw_html.ex

```diff
@@ -70,11 +85,11 @@ defmodule Floki.RawHTML do
   defp build_attrs({attr, value}, attrs), do: ~s(#{attrs} #{attr}="#{value}")
   defp build_attrs(attr, attrs), do: "#{attrs} #{attr}"
 
-  defp tag_for(type, attrs, content) when type in ["script", "style"] do
+  defp tag_for(type, attrs, content, _encoder) when type in ["script", "style"] do
     tag_with_attrs(type, attrs, []) <> Enum.join(content) <> "</#{type}>"
   end
 
-  defp tag_for(type, attrs, children) do
-    tag_with_attrs(type, attrs, children) <> raw_html(children) <> close_end_tag(type, children)
+  defp tag_for(type, attrs, children, encoder) do
```

Comment:
I don't love the 4 arity, but it seemed cleaner than shoving everything inside `build_raw_html`